aspnet-framework/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# escape=`
+
+ARG DOTNET_VERSION=4.8
+FROM mcr.microsoft.com/dotnet/framework/aspnet:${DOTNET_VERSION}
```

Comment:
Does this Dockerfile handle compilation, is the developer supposed to do the compilation first, and can this handle rebuilds?

Reply:
This only runs IIS with a single application at the root, bound via a docker volume mount. It doesn't handle any compilation on its own, initial or otherwise, so the workflow for the developer is the same as Start-Dev (write code, build project in IDE/command line).

It works well for me because Rider can compile the project using the Mono compiler and the output assemblies can be run natively on the .NET Framework.

Reply:
Oh, I didn't know you could build .NET Framework projects in Rider on non-Windows machines.  Nice.

Reply:
Yeah, I just discovered that recently. I was pleasantly surprised that it worked.